admin/goals/index.html

```diff
@@ -9,6 +9,17 @@
   <link rel="shortcut icon" type="image/ico" href="../../favicon.ico">
 </head>
 <body>
+  <header id="navbar">
+    <ul>
+      <li><a href="/">Home</a></li>
+      <li><a href="admin/">Admin</a></li>
+      <li><a href="bootstrap/">Developer Bootstrap</a></li>
+      <li><a href="vpn/">VPN Setup</a></li>
+      <li><a href="ci/">Curriculum</a></li>
+      <li><a href="fll/">FLL</a></li>
+      <li><a href="archives/">Archives</a></li>
```

Comment:
These relative hrefs are incorrect. For example, `admin/` will go to `https://frc3512.github.io/admin/goals/admin`. The same problem applies to all the others. You'll need to prefix them with `../`.